ci-operator/config/openshift/oc/openshift-oc-master.yaml

```diff
@@ -43,6 +43,12 @@ images:
   dockerfile_path: Dockerfile.rhel
   from: cli
   to: recycler
+- from: base
+  to: force-rpms-to-be-built
+  inputs:
+    rpms:
```

Comment:
this one seems to work https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/4855/rehearse-4855-pull-ci-openshift-oc-master-images/2#0:build-log.txt%3A21